### PR TITLE
web: resource specific log routes

### PR DESCRIPTION
**Pull request by jazzdan**
I'd like to:
* Add some more types
* Add a test

But this solves the problem for now.

### CODE REVIEW

File: web/src/HUD.tsx [hunk 1]
```diff
@@ -59,11 +71,21 @@ class HUD extends Component {
...
     }
 
+    let LogsRoute = (props: HudProps) => {
+      let name = props.match.params ? props.match.params.name : ""
+      let logs = ""
+      if (name !== "") {
+        let r = view.Resources.find(r => r.Name === name)
+        logs = r ? r.CombinedLog : ""
+      }
+       return <LogPane log={logs} />
```
Comment (nicks) [inline web/src/HUD.tsx]: weird indentation?

File: web/src/HUD.tsx [hunk 2]
```diff
@@ -73,6 +95,10 @@ class HUD extends Component {
         </Switch>
 
         <Statusbar items={statusItems} toggleSidebar={this.toggleSidebar}  />
+        <Switch>
```
Comment (nicks) [inline web/src/HUD.tsx]: shouldn't this replace the LogPane down below?

Comment (nicks) [reply web/src/HUD.tsx]: or just merge this `Switch` into that `Switch`

File: web/src/HUD.tsx (hunk 1 above)
Comment (jazzdan) [reply web/src/HUD.tsx]: Good catch

**Pull request merged**